scripts/syn.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# Syn
+echo "Building Syn"
+
+HOME_DIR=$1
+
+if [ -f "$HOME_DIR/islandora/configs/variables" ]; then
+  . "$HOME_DIR"/islandora/configs/variables
+fi
+
+cd "$HOME_DIR"
+git clone https://github.com/jonathangreen/Syn.git
```

Comment:
Just marking this so I don't forget. This needs to be updated when the java code moves into the Islandora GitHub.